.github/workflows/quality-checks.yaml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 on:
+  workflow_dispatch:
```

Comment:
Have you tested that this works or does it need to be merged before the Run Worfklow dropdown appears in the Actions tab?

Reply:
I didn't test it here, but I used it for Bellman and it works without issues like this,,,

Reply:
Happy for you to merge and find out (once you've removed the change to deploy.yaml)